packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -465,15 +465,19 @@ - (void)keyDown:(NSEvent *)event
 		viewToFocus = [self nextViewToFocusWithFallback:action];
 	}
 
-	if (!passthrough && viewToFocus != nil)
+	if (passthrough)
+	{
+		[super keyDown:event];
+	}
+	else if (viewToFocus != nil)
 	{
 		[[self window] makeFirstResponder:viewToFocus];
 		[viewToFocus scrollRectToVisible:[viewToFocus bounds]];
 	}
-	// Call super only if there are views to focus
-	else if (viewToFocus != nil)
+	else
 	{
-		[super keyDown:event];
+		// There are no views to pass focus to and we don't want to call `[super keyDown:event]`.
```

Comment:
to me empty else block doesn't make things look clean and concise.

Reply:
I can just delete the else altogether. I agree that looks better but I want it to be explicit that there are _some_ cases where we neither handle the event or call super. I'll take a better idea of refactoring too if anyone has one.

Reply:
Found a way to keep the comment, remove the empty block, and still fix regression.